src/mainwindow.cpp

```diff
@@ -208,8 +213,6 @@ MainWindow::MainWindow(QWidget *parent)
                          QStringLiteral(VERSION) + QStringLiteral(" - build ") +
                          QString::number(BUILD));
 
-    qApp->setProperty("mainWindow", QVariant::fromValue<MainWindow *>(this));
```

Comment:
Why should this be removed? This is used quite regularly in QON!

Reply:
There's a replacement. See instance()

Reply:
And you did use the replacement on all occasions that currently depend on the property? 😅

Reply:
Why does anything depend on the property directly? We provide an interface to access mainWindow, no?

Reply:
Found one direct usage of `qApp->property("mainWindow")` and replaced it.